Index: tegra_multimedia_api/samples/Rules.mk
===================================================================
--- tegra_multimedia_api.orig/samples/Rules.mk
+++ tegra_multimedia_api/samples/Rules.mk
@@ -28,15 +28,6 @@
 #
 ###############################################################################
 
-# Clear the flags from env
-CPPFLAGS :=
-LDFLAGS :=
-
-# ARM ABI of the target platform
-ifeq ($(TEGRA_ARMABI),)
-TEGRA_ARMABI ?= aarch64-linux-gnu
-endif
-
 # Location of the target rootfs
 ifeq ($(shell uname -m), aarch64)
 TARGET_ROOTFS :=
@@ -55,41 +46,12 @@ CLASS_DIR 	:= $(TOP_DIR)/samples/common/
 ALGO_CUDA_DIR 	:= $(TOP_DIR)/samples/common/algorithm/cuda
 ALGO_TRT_DIR 	:= $(TOP_DIR)/samples/common/algorithm/trt
 
-ifeq ($(shell uname -m), aarch64)
-CROSS_COMPILE :=
-else
-CROSS_COMPILE := aarch64-unknown-linux-gnu-
-endif
-AS             = $(CROSS_COMPILE)as
-LD             = $(CROSS_COMPILE)ld
-CC             = $(CROSS_COMPILE)gcc
-CPP            = $(CROSS_COMPILE)g++
-AR             = $(CROSS_COMPILE)ar
-NM             = $(CROSS_COMPILE)nm
-STRIP          = $(CROSS_COMPILE)strip
-OBJCOPY        = $(CROSS_COMPILE)objcopy
-OBJDUMP        = $(CROSS_COMPILE)objdump
-NVCC           = $(CUDA_PATH)/bin/nvcc -ccbin $(CPP)
-
-# Specify the logical root directory for headers and libraries.
-ifneq ($(TARGET_ROOTFS),)
-CPPFLAGS += --sysroot=$(TARGET_ROOTFS)
-LDFLAGS += \
-	-Wl,-rpath-link=$(TARGET_ROOTFS)/lib/$(TEGRA_ARMABI) \
-	-Wl,-rpath-link=$(TARGET_ROOTFS)/usr/lib/$(TEGRA_ARMABI) \
-	-Wl,-rpath-link=$(TARGET_ROOTFS)/usr/lib/$(TEGRA_ARMABI)/tegra
-endif
 
 CPPFLAGS += \
-	-I"$(TARGET_ROOTFS)/usr/include/$(TEGRA_ARMABI)" \
 	-I"$(TOP_DIR)/include" \
 	-I"$(TOP_DIR)/include/libjpeg-8b" \
 	-I"$(TARGET_ROOTFS)/usr/include/libdrm"
 
 LDFLAGS += \
-	-L"$(TARGET_ROOTFS)/usr/lib/$(TEGRA_ARMABI)" \
-	-L"$(TARGET_ROOTFS)/usr/lib/$(TEGRA_ARMABI)/tegra"
-
-LDFLAGS += \
 	-lpthread -lv4l2 -lEGL -lGLESv2 -lX11 \
 	-lnvbuf_utils -lnvjpeg -lnvosd -ldrm
