Index: tegra_multimedia_api/tools/ConvertCaffeToGieModel/Makefile
===================================================================
--- tegra_multimedia_api.orig/tools/ConvertCaffeToGieModel/Makefile
+++ tegra_multimedia_api/tools/ConvertCaffeToGieModel/Makefile
@@ -28,16 +28,7 @@
 #
 ###############################################################################
 
-CC = aarch64-linux-gnu-g++
-
-INCPATHS = -I/usr/include/aarch64-linux-gnu/
-CFLAGS = -Wall -std=c++11 $(INCPATHS)
-
-CFLAGS += -D ENABLE_GIE
-
-LIBPATHS = -L/usr/lib/aarch64-linux-gnu/
-LIBS = -lnvinfer -lnvcaffe_parser
-LFLAGS = $(LIBPATHS) -Wl,--start-group $(LIBS) -Wl,--end-group
+LIBS = -Wl,--start-group -lnvinfer -lnvcaffe_parser -Wl,--end-group
 
 SRCFILES = ConvertCaffeToGieModel_main.cpp
 OUTNAME = ConvertCaffeToGieModel
@@ -45,7 +36,7 @@ OUTNAME = ConvertCaffeToGieModel
 all: $(OUTNAME)
 
 $(OUTNAME) : $(SRCFILES)
-	$(CC) -o $@ $^ $(CFLAGS) $(LFLAGS)
+	$(CXX) -o $@ $^ -DENABLE_GIE $(CPPFLAGS) $(LIBS) $(LDFLAGS)
 
 clean:
 	rm -rf $(OUTNAME)
