Index: git/src/basic/random-util.c
===================================================================
--- git.orig/src/basic/random-util.c
+++ git/src/basic/random-util.c
@@ -36,8 +36,14 @@
 #include "random-util.h"
 #include "time-util.h"
 
+#ifdef AVOID_GETRANDOM
+#define have_syscall_DEFAULT 0
+#else
+#define have_syscall_DEFAULT (-1)
+#endif
+
 int dev_urandom(void *p, size_t n) {
-        static int have_syscall = -1;
+        static int have_syscall = have_syscall_DEFAULT;
 
         _cleanup_close_ int fd = -1;
         int r;
